facebook/delphiFacebook/R/responses.R

```diff
@@ -382,6 +387,7 @@ create_complete_responses <- function(input_data, county_crosswalk)
     "V1", "V2", "V3", "V4_1", "V4_2", "V4_3", "V4_4", "V4_5", # added in Wave 6
     "V9", # added in Wave 7,
     "V2a", "V5a", "V5b", "V5c", "V5d", "V6", "D11", # added in Wave 8
+    "V11", "V12", "V13", "V14_1", "V14_2", # added in Wave 10
```

Comment:
We're also going to need C13b and C13c added. Because we're changing that item to be specifically about indoor activities, we've renamed it.